scripts/pointSelection/balancedAcceptance.yml

```diff
@@ -0,0 +1,26 @@
+script: balancedAcceptance.jl
+description: Chooses sampling points from a priority map.
+
+inputs:
+    priority_map:
+        label: Priority map
+        description: Priority map that Balanced-Acceptance Sampling will run on
+        type: image/tiff;application=geotiff
+        #example: "./foo/bar/priority_map.tiff"
+        example: "/output/pointSelection/combineLayers_jl/73492edb740412f10ee367bda95eb04e/data/priority_map.tiff"
+    bias_toward_high_priority:
+        label: Bias toward high priority
+        description: A real number between 0 and infty
+        type: float
+        example: 3.14159
+    num_points:
+        label: Number of points to select
+        description: Number of points to select
+        type: integer
+        example: 50
+outputs:
+    points:
```

Comment:
Just a suggestion: since the choice of sites is a bit arbitrary, though balanced, it could be cool to ask for a few of these maps so we could choose which map makes more sense... Input param number of maps, output would be an array of geojson files.